sql/core/src/main/scala/org/apache/spark/sql/sources/ddl.scala

```diff
@@ -231,6 +241,32 @@ private [sql] case class CreateTempTableUsing(
   }
 }
 
+private[sql] case class DropTable(
+    tableName: String,
+    isExists: Boolean,
+    temporary: Boolean) extends Command
```

Comment:
@liancheng Thanks for the code review, most of the them are very valueable for me. 
And also I tried your suggestions, but it's not easy to cover both `sql` and `hivesql` dialect test code. And I think to use a logical node and use each dialect's own strategy to convert will more reasonable and easy to understand.  Sorry if I'm wrong.